tests/functional/modules/test_zos_lineinfile_func.py

```diff
@@ -17,10 +17,12 @@
 import re
 import pytest
 import inspect
+import os
```

Comment:
GH is marking `import os` as a new line, but I don't see it being used in the tests, if not used please remove.

Reply:
done